packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -369,8 +374,32 @@ class AppDomain extends Domain {
         ipv6: ipv6,
       );
     }
-
-    final AppInstance app = new AppInstance(_getNewAppId(), runner: runner, logToStdout: daemon.logToStdout);
+    
+    return launch(
+        runner,
+        ({
```

Comment:
formatting looks a little wonky here - perhaps have the `({` on the previous line?

Reply:
I think I've improved it... dartfmt did a bad job with it and I couldn't make it look nice. Hopefully it's better now.

Reply:
dartfmt does this even with that:

<img width="661" alt="screen shot 2018-07-12 at 5 33 39 pm" src="https://user-images.githubusercontent.com/1078012/42646761-c1b5e6ba-85f9-11e8-8846-103a94628583.png">

Here's how it looks as committed now:

<img width="616" alt="screen shot 2018-07-12 at 5 34 18 pm" src="https://user-images.githubusercontent.com/1078012/42646795-ddd8a62a-85f9-11e8-9cf4-93a1de1bfd77.png">

I considered this but it's *way* over 80 chars!

<img width="897" alt="screen shot 2018-07-12 at 5 35 52 pm" src="https://user-images.githubusercontent.com/1078012/42646872-12be0fc4-85fa-11e8-834f-9ce02fa142a9.png">